lib/aliquot.rb

```diff
@@ -50,15 +50,15 @@ def self.start_key_updater(logger)
             resp = conn.get
 
             raise 'Unable to update keys: ' + resp.data[:status_line] unless resp.status == 200
-            cache_control = resp.headers['Cache-Control'].split(/,\s*/)
+            cache_control = resp.headers['Cache-Control'].to_s.split(/,\s*/)
```

Comment:
Hmm. Is this just to make sure that it is a string?

Reply:
Or in case of missing headers?

Reply:
In case the header is missing. The lookup in the hash will be `nil` in that case.

I suspect, however, that some of the subsequent parsing of the header's value will also fail, if the header is missing. I have not looked properly into that.

Reply:
If no match is found, it defaults to every 12 hours, I checked that part :+1:

Reply:
Roger. However, they could at some point change their mind and leave out the header. Assuming they will not change their mind is not very defensive :slightly_smiling_face: 

Somewhat :bike: :house:

Reply:
I completely agree that they might leave out the header. With your "fix", it will default to 12 hours between refreshes.
We should log an error, however, if the header is removed.

Reply:
Uhm. I did not think my fix changed the behaviour when the header is present? Is it not already a string in that case?

Reply:
No, it only changes it when the header is not present.